cpp/src/parquet/printer.cc

```diff
@@ -64,6 +64,25 @@ void PrintPageEncodingStats(std::ostream& stream,
 // the fixed initial size is just for an example
 #define COL_WIDTH 30
 
+void Put(std::ostream& stream, char c, int n) {
```

Comment:
Can you give this a more distinctive name, such as `PutChars`?

Reply:
done